web/modules/custom/do_base/src/Hook/PreprocessParagraphHook.php

```
@@ -0,0 +1,47 @@
+  public function preprocessParagraph(array &$variables): void {
+    $paragraph = $variables['paragraph'] ?? NULL;
+
+    if (!$paragraph instanceof Paragraph || $paragraph->bundle() !== 'civictheme_automated_list') {
+      return;
+    }
+
+    if (!$paragraph->hasField('field_c_p_list_topics') || $paragraph->field_c_p_list_topics->isEmpty()) {
+      return;
+    }
+
+    $topic_ids = array_column($paragraph->field_c_p_list_topics->getValue(), 'target_id');
```

PHPMD (2.15.0)

[error] The variable $topic_ids is not named in camelCase. (undefined)

(CamelCaseVariableName)